pkgs/development/python-modules/fastchat/default.nix

```
@@ -0,0 +1,100 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, aiohttp
+, fastapi
+, httpx
+, markdown2
+, nh3
+, numpy
+, prompt-toolkit
+, pydantic
+, requests
+, rich
+, shortuuid
+, tiktoken
+, uvicorn
+, anthropic
+, openai
+, ray
+, wandb
+, einops
+, gradio
+, accelerate
+, peft
+, sentencepiece
+, torch
+, transformers
+, protobuf
+}:
+let
+  version = "0.2.28";
+in
+buildPythonPackage {
+  pname = "fast-chat";
```

```suggestion
  pname = "fschat";
```
The pname should be the same as pypi's name.
https://pypi.org/project/fschat/